experimenter/tests/integration/nimbus/test_desktop_targeting.py

```diff
@@ -32,7 +39,7 @@ def test_check_advanced_targeting(
 
     # Inject filter expression
     selenium.get("about:blank")
-    with Path("nimbus", "utils", "filter_expression.js").open() as js:
+    with Path(filter_expression_path).open() as js:
```

Comment:
`filter_expression_path` is already a `Path`, so you can call `.open()` directly.